README.md

```diff
@@ -21,35 +28,45 @@ In addition to tests, this package creates the model `int_all_dag_relationships`
 
   ### Cloning via local packages
 
-    1. Clone [repository](https://github.com/dbt-labs/dbt-project-evaluator) locally via normal git workflow
-    2. Add package to your `packages.yml` in your project:
-        
-        ```yaml
-        # in packages.yml
-        
-        packages:
-          - local: <path/to/package> # use a local path
-        ```
-    3. Run `dbt deps` to install
-    4. Execute a `dbt build --select package:dbt_project_evaluator`!
+  1. Clone the [repository](https://github.com/dbt-labs/dbt-project-evaluator) locally via normal git workflow
+  2. Add package to your `packages.yml` in your project:
+      
+      ```yaml
+      # in packages.yml
+      
+      packages:
+        - local: <path/to/package> # use a local path
+      ```
+  3. Run `dbt deps` to install
+  4. Execute a `dbt build --select package:dbt_project_evaluator`!
 
   ### Cloning via git address
 
-    1. Add package to your `packages.yml` in your project:
-        
-        ```yaml
-        # in packages.yml
-        
-        packages:
-          - git: "https://github.com/dbt-labs/dbt-project-evaluator.git"
-            revision: v0.1.0
-        ```
-        
-    2. Run `dbt deps` to install
-    3. Execute a `dbt build --select package:dbt_project_evaluator`!
+  1. Add package to your `packages.yml` in your project:
+      
+      ```yaml
+      # in packages.yml
+      
+      packages:
+        - git: "https://github.com/dbt-labs/dbt-project-evaluator.git"
+          revision: v0.1.0
+      ```
+      
+  2. Run `dbt deps` to install
+  3. Execute a `dbt build --select package:dbt_project_evaluator`!
     
+  ### Additional setup for Databricks/Spark
 
+  In your `dbt_project.yml`, add the following config:
+  ```yaml
+  dispatch:
+    - macro_namespace: dbt_utils
+      search_order: ['dbt_project_evaluator', 'spark_utils', 'dbt_utils']
+  ```
+
+  This is required because the project currently provides limited support for arrays macros for Databricks/Spark which is not part of `spark_utils` yet.
```

Comment:
Should we open up an issue to move the support for these array macros into `spark_utils`?